idx/idx.go

```diff
@@ -147,6 +147,9 @@ type MetricIndex interface {
 	// LastUpdate time is >= the given value.
 	Tags(int, string, int64) ([]string, error)
 
+	AutoCompleteTags(int, string, []string, int64, uint) ([]string, error)
```

Comment:
can you document these functions here? (I know you documented the memory implementations, but typically the docs people see when navigating the code is the interface methods)

Reply:
sure https://github.com/grafana/metrictank/pull/779/commits/818e0b2a1b5d2631dab2c210c530b1c0c732215b